fitbenchmarking/controllers/mantid_controller.py

```diff
@@ -231,9 +231,18 @@ def fit(self):
         """
         Run problem with Mantid.
         """
+        minimizer_str = self.minimizer
+        if self.minimizer == 'FABADA':
+            # The max iterations needs to be larger for FABADA
+            # to work; setting to the value in the mantid docs
+            minimizer_str += (",Chain Length=100000"
+                              ",Steps between values=10"
+                              ",Convergence Criteria=0.01")
+            self._added_args['MaxIterations'] = 2e6
```

Comment:
Apparently 2e6 is float so need to use the number
```suggestion
            self._added_args['MaxIterations'] = 2000000
```